public/js/routes/line/index.js

```diff
@@ -7,62 +10,66 @@ var TripsCollection = require( '../../collections/trips' );
 var LineStatusModel = require( '../../models/line-status' );
 
 var data = require( '../../data' );
-var setTitle = require( '../../lib/set-title' );
 
-function lineOverviewRoute( lineSlug ) {
-  /* jshint validthis: true */
+module.exports = {
+  url: '^/:line(red|orange|blue)',
 
-  // Error out early if the route didn't get a valid line slug
-  var invalidLineSlug = [ 'red', 'orange', 'blue' ].indexOf( lineSlug ) < 0;
+  update: function( opts ) {
+    this.enter( opts );
+  },
```

Comment:
Interesting. I think what I have is sufficient for now: I am not using the nesting feature of the router at all, so this solution, while slightly hacky, should allow me to get by until I decide to strip it out and use another router later on :)